.github/scripts/auto-label.cjs

```diff
@@ -14,28 +14,43 @@ const github = require('@actions/github');
 const babelParser = require('recast/parsers/babel');
 const { HttpClient } = require('@actions/http-client');
 
-const prefixLabelMap = {
-  'resources/': 'resources',
-  'util/': 'util',
-  'test/': 'test',
-  '.mocharc.cjs': 'test',
-  'eslint/': 'style',
-  '.eslintrc.js': 'style',
-  '.github/workflows/': 'ci',
-  '.github/scripts/': 'ci',
-  'ui/config/': 'config',
-  'ui/dps/': 'dps',
-  'ui/eureka/': 'eureka',
-  'ui/fisher/': 'fishing🎣',
-  'ui/jobs/': 'jobs',
-  'ui/oopsyraidsy/': 'oopsy',
-  'ui/pullcounter/': 'pullcounter',
-  'ui/radar/': 'radar',
-  'ui/test/': 'test',
-  'ui/raidboss/raidemulator': 'raidemulator',
-  'ui/raidboss/emulator': 'raidemulator',
+const langToLabel = (lang) => `💬${lang.toUpperCase()}`;
+
+// Only the first match applies.
+const regexLabelMap = {
+  '^docs/ko-KR/': ['docs', langToLabel('ko')],
+  '^docs/zh-CN/': ['docs', langToLabel('cn')],
+  '^docs/zh-TW/': ['docs', langToLabel('cn')],
+  '.*\\.md$': ['docs'],
+  '^docs/': ['docs'],
+  '^screenshots/': ['docs'],
+  '^resources/': ['resources'],
```

Comment:
Triple legend nit: timerbar_test.html is in resources.

Reply:
I think we should move timerbar_test to some test/manual/ directory then?

Reply:
`test/danakj_timecapsule/`